### ISSUE TITLE
ActionMailbox direct e-mail replies

### ISSUE THREAD

**Issue by andreibondarev** [thredded/thredded#855]
For one of my project, I'm implementing direct e-mail replies by using the new ActionMailbox framework. The recipient replies directly to the Thredded e-mail notification and that post gets automatically created in the topic. What is the appetite for having this functionality live in the `thredded` gem itself?

**Comment by glebm** [comment]
This would be very cool!

**Comment by jayroh** [comment]
I had had this in an OLD OLD version of this codebase and that implementation was slapped together with spit and duct tape.

Would be great to see this with a nice abstraction like ActionMailbox :)

**Comment by Keerthi1244** [comment]
Yeah, that could be awesome. [indijobs4u](http://www.indijobs4u.in)

**Comment by seabornhq** [comment]
@andreibondarev @glebm may take a shot at this?
